layouts/partials/versionnavigation.html

```diff
@@ -1,6 +1,15 @@
-<!-- templates -->
-{{- define "version-nav" }}
-  <option value="/gloo/latest{{ .RelPermalink}}" {{if in .RelPermalink "latest/"}} selected{{end}}>
+<!--<div title="v{{.Site.Data.Solo.CodeVersion}}"
+            style="overflow: hidden;
+            text-overflow: ellipsis;
+            white-space: nowrap;"
+        >
+          {{ .Site.Data.Solo.CodeVersion }}
+        </div>-->
+  
+<!-- Need to run throuugh a RANGE of .Site.Data.Solo.DocsVersions .  This is tough to test locally as the version folders aren't generated -->
+<label for="version-selection" id="version-selection-label">Version:</label>
+<select id="version-selection" onchange="javascript:location.href = this.value;">
+  <option value="/latest/{{ .RelPermalink}}" {{if in .RelPermalink "latest/"}} selected{{end}}>
```

Comment:
i dont' think we want the trailing slash, i.e.:
```suggestion
  <option value="/latest{{ .RelPermalink}}" {{if in .RelPermalink "latest/"}} selected{{end}}>
```

Reply:
I think we do?  And just tested it live (as this isn't really new code. More just moved around).

Reply:
This is the version currently deployed: https://github.com/solo-io/hugo-theme-soloio/blob/master/layouts/partials/flex/versionnavigation.html#L3 , which works as expected and doesn't render an extra `/`

Reply:
that file wasn't actually previously being used.

Reply:
oops this is the one i meant to link: https://github.com/solo-io/hugo-theme-soloio/blob/master/layouts/partials/flex/body-beforecontent.html#L23